enterprise/backend/src/metabase_enterprise/api/routes.clj

```diff
@@ -47,4 +48,7 @@
     (ee.api.common/+require-premium-feature :serialization (deferred-tru "Serialization") api.serialization/routes))
    (context
     "/autodescribe" []
-    (ee.api.common/+require-premium-feature :llm-autodescription (deferred-tru "LLM Auto-description") llm.api/routes))))
+    (ee.api.common/+require-premium-feature :llm-autodescription (deferred-tru "LLM Auto-description") llm.api/routes))
+   (context
+    "/uploads" []
+    (ee.api.common/+require-premium-feature :uploads (deferred-tru "Upload Maintenance") api.uploads/routes))))
```

Comment:
I'm not sure who needs to sign off on the feature name and title, I suspect we'll live with this decision for a long time.